packages/nc-gui/composables/useData.ts

```diff
@@ -732,7 +732,7 @@ export function useData(args: {
 
     await callbacks?.syncCount?.()
     await callbacks?.syncPagination?.()
-    await callbacks?.globalCallback?.()
+    callbacks?.globalCallback?.()
```

Comment:
What is the purpose of this change? This is a shared composable so is changing callback logic a good idea?

Reply:
The type definition of `globalCallback` is `globalCallback?: (...args: any[]) => void`. It is not async. Removed the `await` to handle TS warning. I think the `await` was there by mistake.

Reply:
Please keep await and replace type definition to expect a promise instead, having await on a non-promise will not have any significant difference but counter-part can cause unexpected results (as we await wrapper function everything should be handled before going to next step)